yt_dlp/extractor/wetv.py

```diff
@@ -0,0 +1,197 @@
+            'title': 'EP1: You Are My Glory',
+            'description': 'md5:831363a4c3b4d7615e1f3854be3a123b',
+        },
+    }, {
+        'url': 'https://wetv.vip/en/play/lcxgwod5hapghvw-WeTV-PICK-A-BOO/i0042y00lxp-Zhao-Lusi-Describes-The-First-Experiences-She-Had-In-Who-Rules-The-World-%7C-WeTV-PICK-A-BOO',
+        'md5': '71133f5c2d5d6cad3427e1b010488280',
+        'info_dict': {
+            'id': 'i0042y00lxp',
+            'ext': 'mp4',
+            'title': 'md5:f7a0857dbe5fbbe2e7ad630b92b54e6a',
+            'description': 'md5:76260cb9cdc0ef76826d7ca9d92fadfa',
+        },
+    }]
+
+    def _extract_video_formats_and_subtitles(self, video_id, api_response):
+        video_response = api_response['vl']['vi'][0]
+
+        formats, subtitles = [], {}
+        for video_format in video_response['ul']['ui']:
+            if video_format.get('hls'):
+                fmts, subs = self._extract_m3u8_formats_and_subtitles(
+                    video_format['url'] + video_format['hls']['pname'], video_id, 'mp4')
+
+                formats.extend(fmts)
+                self._merge_subtitles(subs, target=subtitles)
+            else:
+                formats.append({
+                    'url': f"{video_format['url']}{video_response['fn']}?vkey={video_response['fvkey']}",
+                    'width': video_response.get('vw'),
+                    'height': video_response.get('vh'),
+                    'ext': 'mp4',
+                })
+
+        return formats, subtitles
+
+    def _extract_video_subtitles(self, api_response, subtitles_format):
+        subtitles = {}
+        for subtitle in api_response['sfl']['fi']:
+            subtitles[subtitle['lang'].lower()] = [{
+                'url': subtitle['url'],
+                'ext': subtitles_format,
+                'protocol': 'm3u8_native' if subtitles_format == 'vtt' else 'http'
+            }]
+
+        return subtitles
+
+    def _real_extract(self, url):
+        video_id, series_id = self._match_valid_url(url).group('id', 'series_id')
+        webpage = self._download_webpage(url, video_id)
+
+        formats, subtitles = [], {}
+        for video_format, subtitle_format in (('mp4', 'srt'), ('hls', 'vtt')):
+            api_response = self._get_video_api_response(url, video_id, series_id, subtitle_format, video_format)
+
+            fmts, subs = self._extract_video_formats_and_subtitles(video_id, api_response)
+            native_subtitles = self._extract_video_subtitles(api_response, subtitle_format)
+
+            formats.extend(fmts)
+            self._merge_subtitles(native_subtitles, target=subtitles)
+
+        return {
+            'id': video_id,
+            'title': self._og_search_title(webpage),
+            'description': self._og_search_description(webpage),
+            'formats': formats,
+            'subtitles': subtitles,
+        }
+
+
+class WeTvSeriesIE(WeTvBaseIE):
+    _VALID_URL = WeTvBaseIE._VALID_URL_BASE + r'/(?P<id>\w+)(?:-[^?#]+)?'
```

Comment:
```suggestion
    _VALID_URL = WeTvBaseIE._VALID_URL_BASE + r'/(?P<id>\w+)(?:-[^?#]+)?/?(?:[?#]|$)'
```
should fix tests

Reply:
Fix series url

Reply:
It didn't fix it mm

Reply:
Nvm, it was just missing a / 😅 , thanks!